package.json

```diff
@@ -42,6 +42,44 @@
         "libusb-1.0-0-dev",
         "libyaml-dev"
       ]
+    },
+    "builder": {
+      "appId": "io.resin.etcher",
+      "copyright": "Copyright 2016 Resinio Ltd",
+      "productName": "Etcher",
+      "mac": {
+        "category": "public.app-category.developer-tools"
+      },
+      "dmg": {
+        "iconSize": 110,
+        "contents": [
+          {
+            "x": 140,
+            "y": 225
+          },
+          {
+            "x": 415,
+            "y": 225,
+            "type": "link",
+            "path": "/Applications"
+          }
+        ],
+        "window": {
+          "width": 540,
+          "height": 405
+        }
+      },
+      "linux": {
+        "category": "Utility",
+        "packageCategory": "utils",
+        "synopsis": "Etcher is a powerful OS image flasher built with web technologies to ensure flashing an SDCard or USB drive is a pleasant and safe experience. It protects you from accidentally writing to your hard-drives, ensures every byte of data was written correctly and much more."
+      },
+      "deb": {
```

Comment:
Should there also be an `rpm` section?

Reply:
There were some known issues with Etcher's RPM dependencies, but I can't remember what it was. I think RPM repositories didn't support any elevation package we used, or something like that. Maybe worth trying to add it back in a new PR?

Reply:
> I think RPM repositories didn't support any elevation package we used, or something like that.

I think that was the "elevate at startup" PR that @Shou was working on, which didn't get merged in the end? The current Etcher (Makefile & electron-builder) setup builds both deb and rpm files, so IMHO we shouldn't drop rpm files when switching to concourse.